sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala

```diff
@@ -1345,7 +1371,9 @@ abstract class DynamicPartitionPruningSuiteBase
     }
   }
 
-  test("SPARK-32817: DPP throws error when the broadcast side is empty") {
+  test("SPARK-32817: DPP throws error when the broadcast side is empty",
+    DisableAdaptiveExecution("EliminateJoinToEmptyRelation " +
```

Comment:
We can disable this rule by setting `ADAPTIVE_OPTIMIZER_EXCLUDED_RULES`.

Reply:
Updated.